trymito.io/pages/excel-to-python/[...slug].tsx

```diff
@@ -138,11 +138,11 @@ const ExcelToPythonGlossaryPage = (props: {pageContent: PageContent}) => {
               {/* Understanding the Excel Function */}
               <section className={excelToPythonStyles.section}>
                   <h2 
-                    id={`Understanding Excel's ${functionNameShort} Function`}
+                    id={`Excel's ${functionNameShort} Function`}
                     className={excelToPythonStyles.link}
                   >
-                    Understanding Excel&apos;s {functionNameShort} Function
-                    <a className={excelToPythonStyles.section_copy} href={`#Understanding Excel's ${functionNameShort} Function`}>#</a>
+                    Understanding the {pageContent.functioNameLong} Function in Excel
```

Comment:
`is functioNameLong` the right name? Can we rename this to `functionNameLong` everywhere if so?